components/Chips/examples/ChipsFieldDeleteEnabledViewController.swift

```diff
@@ -45,11 +35,14 @@ class ChipsFieldDeleteEnabledViewController : UIViewController, MDCChipFieldDele
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    view.backgroundColor = colorScheme.backgroundColor
+    containerScheme.colorScheme = MDCSemanticColorScheme()
+    containerScheme.shapeScheme = MDCShapeScheme()
+    containerScheme.typographyScheme = MDCTypographyScheme()
+    view.backgroundColor = containerScheme.colorScheme!.backgroundColor
     chipField.frame = .zero
     chipField.delegate = self
     chipField.textField.placeholderLabel.text = "This is a chip field."
-    chipField.backgroundColor = colorScheme.surfaceColor
+    chipField.backgroundColor = containerScheme.colorScheme!.surfaceColor
```

Comment:
`colorScheme` is nullable and may be injected as `nil` (even if the container is not). Please add conditional support to fall-back to a default value.  Same for the `view.backgroundColor` above.